Makefile

```diff
@@ -68,7 +68,16 @@ URL_IGNORE=\#$\
 NEW_URLS=$(shell scripts/ignore_new_urls.sh 2> /dev/null)
 URL_IGNORE:=$(URL_IGNORE)$(NEW_URLS)
 
+# Skip validation from (Probably) not released version (last one)
+# Get current version from release notes
+VERSION=$(shell egrep -m1 '\#\#' content/en/news/release-notes.md | sed 's/\#\# //' | sed 's/.0//' | sed 's/\./-/' )
+URL_VERSION=,/^https://v$(VERSION).kiali.io/
+URL_IGNORE:=$(URL_IGNORE)$(URL_VERSION)
+VERSION=$(shell egrep -m2 '\#\#' content/en/news/release-notes.md | sed 's/\#\# //' | sed 's/.0//' | sed 's/\./-/' | tail -n1)
+URL_VERSION=,/^https://v$(VERSION).kiali.io/
+URL_IGNORE:=$(URL_IGNORE)$(URL_VERSION)
+
```

Comment:
This is fine although would it be easier to use sort of a `not equal 'staging'` logic?

Reply:
Good idea, @jshaughn ! I will see how to improve it.